app/Actions/Server/ConfigureTrafficAnalytics.php

```
@@ -0,0 +1,27 @@
+    public function handle(Server $server, bool $enable): void
+    {
+        $server->settings->is_traffic_analytics_enabled = $enable;
+        $server->settings->save();
+        $server->refresh();
+
+        // Regenerate proxy config so the (Traefik) access-log flags / (Caddy) log labels take effect.
+        GetProxyConfiguration::run($server, forceRegenerate: true);
+        RestartProxyJob::dispatch($server);
+
+        // Recreate Sentinel with the traffic env + proxy-log mount (or without, when disabling).
+        StartSentinel::run($server, restart: true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Actions\Server\StartSentinel' in method 'handle'. (undefined)

(StaticAccess)